main.cpp

```diff
@@ -16,8 +16,10 @@ void customLogHandler(QtMsgType type, const QMessageLogContext &context,
 
 int main(int argc, char *argv[])
 {
+	// The version that Ubuntu uses for QT isn't hiqh enough to support this, so neither of these will work.
+	// commenting both of them out will work
 	//QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-	QApplication::setAttribute(Qt::AA_DisableHighDpiScaling);
+	//QApplication::setAttribute(Qt::AA_DisableHighDpiScaling);
```

Comment:
Sadly, we need this line to be in there, because without it, certain features do not work on Mac OSX.  How can we keep this in?